astropy/units/format/ogip.py

```diff
@@ -19,6 +19,7 @@
 import keyword
 import math
 import os
+import copy
```

Comment:
`copy` is only used in `_to_decomposed_alternative()` class method *if* the given unit is a composite unit and it has log scale. Does that situation ever happen?